background.ts

```diff
@@ -560,7 +559,7 @@ ipcMainProxy.on('k8s-integrations', async() => {
 });
 
 ipcMainProxy.on('k8s-integration-set', (event, name, newState) => {
-  console.log(`Setting k8s integration for ${ name } to ${ newState }`);
+  console.debug(`Setting k8s integration for ${ name } to ${ newState }`);
```

Comment:
Sorry to nitpick, but I think this one is unnecessary as well? `name` and `newState` should be logged by the `on` wrapper, along with the `k8s-integration-set` event name, right?

Reply:
Good catch.  Removed.